app/elements/socobo-ranking/socobo-ranking.html

```diff
@@ -169,10 +171,16 @@
           SocoboRanking.getInstance(this.userLogin)
             .getData()
             .then(function(data) {
-              this.notify("Loading Data finished! Calculating started...");
               this._calcRanking(data);
+              // reset loading flag
+              this.isLoading = false;
+              // notify user
+              this.notify("Ranking - Calculating finished!");
```

Comment:
IMO, this one is not required as well. Especially since it is fired wherever you are.